benchmark/common.js

```diff
@@ -203,15 +210,29 @@ Benchmark.prototype.end = function(operations) {
 
 Benchmark.prototype.report = function(value) {
   var heading = this.getHeading();
-  if (!silent)
-    console.log('%s: %s', heading, value.toFixed(5));
+
+  if (!silent) {
+    if (outputFormat == 'default')
+      console.log('%s: %s', heading, value.toFixed(5));
+    else if (outputFormat == 'csv')
+      console.log('%s,%s', heading, value.toFixed(5));
+  }
```

Comment:
Does the combination `silent === true && outputFormat === 'csv'` make sense?  If not, you could write this as:

```javascript
if (!silent && outputFormat == 'default')
  // ...
else if (outputFormat == 'csv')
  // ...
```